### PR TITLE
feat(FTL-8509): add info about active and default project

### PR DESCRIPTION
**Pull request by anusha-kommu**

### CODE REVIEW

**Review by ezgi-affinidi — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -22,10 +22,22 @@ To do that, click on Affinidi logo in sidebar, then click on “Create an accoun
...
 ![Create an Account](./media/docs/create_account.png)
 
-When you’re authenticated, just click on “Create project” button and enter a project’s name. That’s it!  
-We’ll automatically create a digital identity for you – your personal DID.
+Once the account is created, a `Default Project` will be created automatically. As part of it a digital identity will be created for you – your personal DID.
 
-![Project Details](./media/docs/project_details.png)
+![Default Project](./media/docs/default_project.png)
+
+Initially `Default project` will be your `Active Project`.
```
Comment (carlos-affinidi) [inline README.md:29]:
```suggestion
Initially the `Default Project` will be set as your `Active Project`.
```

File: README.md [hunk 2]
```diff
@@ -22,10 +22,22 @@ To do that, click on Affinidi logo in sidebar, then click on “Create an accoun
 
 ![Create an Account](./media/docs/create_account.png)
 
-When you’re authenticated, just click on “Create project” button and enter a project’s name. That’s it!  
-We’ll automatically create a digital identity for you – your personal DID.
+Once the account is created, a `Default Project` will be created automatically. As part of it a digital identity will be created for you – your personal DID.
```
Comment (carlos-affinidi) [inline README.md:25]:
```suggestion
Once the account is created, a project named `Default Project` will be created automatically. As part of it, a digital identity will be created for you – your personal DID.
```

File: README.md [hunk 3]
```diff
@@ -22,10 +22,22 @@ To do that, click on Affinidi logo in sidebar, then click on “Create an accoun
...
 
-![Project Details](./media/docs/project_details.png)
+![Default Project](./media/docs/default_project.png)
+
+Initially `Default project` will be your `Active Project`.
+
+Active Project is the project on which you can perform all the [operations](#features) below. There will be only 1 active project at a time.
```
Comment (carlos-affinidi) [inline README.md:31]:
```suggestion
Your `Active Project` is the project on which you can perform all the [operations](#features) below. There will always be only 1 active project at a time.
```

File: README.md [hunk 4]
```diff
@@ -22,10 +22,22 @@ To do that, click on Affinidi logo in sidebar, then click on “Create an accoun
...
+
+You can also manually create projects by clicking on “Create project” button and enter a project’s name.
```
Comment (carlos-affinidi) [inline README.md:33]:
```suggestion
You can also manually create projects by clicking on `Create Project` button and enter a project’s name.
```

File: README.md [hunk 5]
```diff
@@ -22,10 +22,22 @@ To do that, click on Affinidi logo in sidebar, then click on “Create an accoun
...
+As the default project is the active project, all other projects will be shown under the `inactive projects` folder.
```
Comment (carlos-affinidi) [inline README.md:34]:
```suggestion
As the default project is the active project, all other projects will be shown under the `Inactive Projects` folder.
```

**Review by carlos-affinidi — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (anusha-kommu) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (anusha-kommu) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (anusha-kommu) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (anusha-kommu) [reply README.md]: done

File: README.md [hunk 6]
```diff
@@ -22,10 +22,22 @@ To do that, click on Affinidi logo in sidebar, then click on “Create an accoun
...
+Your `Active Project` is the project on which you can perform all the [operations](#features) below. There will always be only 1 active project at a time.
+
+You can also manually create projects by clicking on `Create Project` button and enter a project’s name.
+As the default project is the active project, all other projects will be shown under the `Inactive Projects` folder.
+
+![Inactive Projects](./media/docs/inactive_projects.png)
+
+You can quickly change the active project any time by clicking on the respective `Activate` icon.
+
+![Activate Project](./media/docs/activate_project.png)
```
Comment (vitalyrudenko-affinidi) [inline README.md:40]: Personally, I think that **bold** here would make more sense than `monospace`, but this is nitpicking

**Review by vitalyrudenko-affinidi — APPROVED** [review]

**Review by carlos-affinidi — APPROVED** [review]

**Pull request merged**